homeassistant/components/number/const.py

```diff
@@ -11,4 +11,7 @@
 
 DOMAIN = "number"
 
+MODE_SLIDER = "slider"
```

Comment:
Minor suggestion: these could be moved to `homeassistant.const`, since `input_number` uses them, too. 

https://github.com/home-assistant/core/blob/14f85d87314bb2bee4226962873f0c9de8034242/homeassistant/components/input_number/__init__.py#L37-L38

Reply:
I didn't want to affect other integrations in this PR.

Reply:
That's fair. I approved the pull request.